forem/forem
# Don't update last_buffered timestamp when creating a suggested tweet

## What type of PR is this? (check all applicable)

## Description
This fixes a bug: when a suggested tweet is made, it updates the article's `last_buffered` timestamp. Based on how we display the article, it makes it seem like the suggested tweet has already been buffered. By not updating the timestamp, we won't incorrectly display that the article has been buffered already.

## Related Issues
Resolves https://github.com/forem/tech-admin/issues/46

## QA Instructions, Screenshots, Recordings
1. Make a suggested tweet of an article you have: localhost:3000/:article_path/manage
2. Check that the article is not displaying a `last_buffered` timestamp
3. Check that there isn't text with orange backgrounds

### UI accessibility concerns?
No

## Added tests?
- [x] No, and this is why: just fixing expectations here. there aren't too many tests for buffer updates and I'd rather have that in a different PR

## Added to documentation?
